Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -12,21 +14,23 @@ public class EntryRenderer : ViewRenderer<Entry, UITextField>
 		UIColor _defaultTextColor;
 		bool _disposed;
 
-		public EntryRenderer()
+		static readonly int basetHeight = 30;
+		static CGSize initialSize = CGSize.Empty;
+
+		public EntryRenderer() 
 		{
 			Frame = new RectangleF(0, 20, 320, 40);
 		}
 
-		public override SizeRequest GetDesiredSize(double widthConstraint, double heightConstraint)
+		public override SizeRequest GetDesiredSize(double widthConstraint, double heightConstraint) 
 		{
-			//with borderStyle set to RoundedRect, iOS always returns a height of 30
-			//https://stackoverflow.com/a/36569247/1063783
-			//we get the current value, and restor it, to allow custom renderers to change the border style
-			var borderStyle = Control.BorderStyle;
-			Control.BorderStyle = UITextBorderStyle.None;
-			var size = Control.GetSizeRequest(widthConstraint, double.PositiveInfinity);
-			Control.BorderStyle = borderStyle;
-			return size;
+			var baseResult = base.GetDesiredSize(widthConstraint, heightConstraint);
+
+			NSString testString = new NSString("Tj");
```

Comment:
Like...Tijuana?

Reply:
nah just using a string that has both a cap and a descender